<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialAssistant</class>
 <widget class="QWidget" name="SerialAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SerialAssistant</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>120</height>
    </size>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QComboBox" name="serialChoose">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>COM1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM3</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="serialBaudRate">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit { background-color: rgba(0, 0, 0, 0); border: none; }</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;波特率&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit { background-color: rgba(0, 0, 0, 0); border: none; }</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;串口选择&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>350</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>120</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="serialLink">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font-size:12pt;
	font-family: Arial;
    font-weight: bold;
    /* font-style: italic; */
}</string>
      </property>
      <property name="text">
       <string>连接串口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="serialBreak">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font-size:12pt;
	font-family: Arial;
    font-weight: bold;
    /* font-style: italic; */
}</string>
      </property>
      <property name="text">
       <string>断开串口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="serialRefresh">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font-size:12pt;
	font-family: Arial;
    font-weight: bold;
    /* font-style: italic; */
}</string>
      </property>
      <property name="text">
       <string>刷新串口</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>720</width>
     <height>420</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>720</width>
     <height>420</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>720</width>
     <height>420</height>
    </size>
   </property>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>720</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>720</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>60</height>
     </size>
    </property>
    <widget class="QTextEdit" name="textEdit_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>104</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit { background-color: rgba(0, 0, 0, 0); border: none; }</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;发送区&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_4">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>104</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit { background-color: rgba(0, 0, 0, 0); border: none; }</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;接收区&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>721</width>
      <height>321</height>
     </rect>
    </property>
    <widget class="QPlainTextEdit" name="sendInfo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>330</width>
       <height>320</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>330</width>
       <height>320</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>320</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit {
	font-size:12pt;
}</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="receiveInfo">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>330</width>
       <height>320</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>330</width>
       <height>320</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>320</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit {
	font-size:12pt;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_8" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>720</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>720</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>30</height>
     </size>
    </property>
    <widget class="QPushButton" name="serialInfoClear">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>0</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="serialBySendInfo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
